addons/l10n_es_edi_tbai_multi_refund/models/account_edi_format.py

```
@@ -0,0 +1,13 @@
+from contextlib import suppress
+
+from odoo import models
+
+
+class AccountEdiFormat(models.Model):
+    _inherit = 'account.edi.format'
+
+    def _l10n_es_tbai_get_in_invoice_values_batuz(self, invoice):
+        values = super()._l10n_es_tbai_get_in_invoice_values_batuz(invoice)
+        with suppress(KeyError):
+            values['credit_note_invoices'] |= invoice.l10n_es_tbai_reversed_ids
```

Would be better to check if the key exists instead of suppressing an exception IMO

---

https://docs.python.org/3/glossary.html#term-EAFP

---

Didn't know about this, thanks!